hazel/third_party/cabal2bazel/bzl/cabal_package.bzl

```diff
@@ -436,7 +446,7 @@ def _get_build_attrs(
                  ]),
         defines = [o[2:] for o in build_info.ccOptions if o.startswith("-D")],
         textual_hdrs = list(headers),
-        deps = ["{}//:rts-headers".format(ghc_workspace)] + select(cdeps) + cc_deps + elibs_targets,
+        deps = [rts_import_name] + select(cdeps) + cc_deps + elibs_targets,
```

Comment:
I'm not sure if this is related to the MacOS segfault. But, a side effect of this is that `HSrts-ghc8.6.4` appears in the `extra-libraries` field of Hazel haskell_library targets.

Generally, in the following setup
```
haskell_library(name = "hs-lib-a", ...)
cc_library(name = "c-lib", deps = [":hs-lib-a"], ...)
haskell_library(name = "hs-lib-b", deps = [":c-lib", "hs-lib-a"], ...)
```
`hs-lib-b`'s package configuration file would list `hs-lib-a` both in `depends` and in `extra-libraries`.